pstar.json

```diff
@@ -4,14 +4,14 @@
 			"maintainer": "vullk4n",
 			"oem": "Motorola",
 			"device": "Moto Edge 20 Pro",
-			"filename": "crDroidAndroid-15.0-20241215-pstar-v11.0-BETA.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/pstar/11.x/crDroidAndroid-15.0-20241215-pstar-v11.0-BETA.zip/download",
-			"timestamp": 1734206249,
-			"md5": "83fdc45fd4824b0fa2139b0132915498",
-			"sha256": "d4443528293b7e5659ff0e8783bc803c0daa6aac015574061c94c01108144f46",
-			"size": 1110056306,
+			"filename": "crDroidAndroid-15.0-20241221-pstar-v11.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/pstar/11.x/crDroidAndroid-15.0-20241221-pstar-v11.0.zip/download",
+			"timestamp": 1734745910,
+			"md5": "d3f0c3a547eadeea2d35f022c1663fc3",
+			"sha256": "bd20d6a4c1cda6e1817e334c78528e5fbf02c6db8275b07ccd04749074c21d9a",
+			"size": 1116919491,
 			"version": "11.0",
-			"buildtype": "BETA",
+			"buildtype": "Monthly",
 			"forum": "https://xdaforums.com/t/rom-official-15-0-moto-edge-20-pro-crdroid-11-beta-monthly-2024-12-15.4708062/",
```

Comment:
fix crDroid Device Group and Support Community link
install link also
and ofc d/l link